build.gradle

```diff
@@ -69,6 +69,7 @@ subprojects {
       testImplementation "org.springframework:spring-test"
       testImplementation "org.hamcrest:hamcrest-core"
       testRuntimeOnly "cglib:cglib-nodep"
+      testRuntimeOnly "org.junit.vintage:junit-vintage-engine"
```

Comment:
Isn't the idea to add
```
test {
   useJunitPlatform()
}
```
also?

Reply:
Updated `useJUnitPlaform()`  in build.gradle and removed this from sub modules.